### DISCUSSION TITLE
2024/02/laoshe/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#99] [General]
# 2024/02/laoshe/

https://xiangyun.rbind.io/2024/02/laoshe/

**Comment by earfanfan** [comment]
>而且还是名列前矛的十大虐心小说之一《当时已枉然》

这个名列前茅的十大是哪个榜单，我完全没听说过这部小说，我记得大学看的言情小说都是叫撒四大言情天后写的，好像是桐华、藤萍、匪我思存。你的感知力好强啊，我当年看虐心言情小说好像还没咋看懂，没怎么被虐到过，倒是气到乐到的时候更多。

>原来，现在纯粹的读书和几个世纪前一样，都是贵族的娱乐。

湘云你说的读书包括读通俗文学么？我记得看过一些记录，许多年前金庸古龙连载武侠小说时，很多人追看。书籍也只是一种信息传递的媒介，而我们国家书籍的价格也都不算贵，微信读书上蛮多免费的，很多网络小说程式上也有很多免费的。

你也看过那本《置身事内》，此书因为把中国政府与经济发展写得比较通俗易懂，豆瓣上有8万人标记读过、1.6万人标记在读、15.5万人标记想读，这本书2021年出版现在排豆瓣图书第56位。当然，人们读这本书大多数应该都是冲这本书揭示的内容或者名头来的，确实应该是带有目的而不是纯粹的娱乐，但我的理解是，我们国家一统体制下政府的政策对平民影响太大，所以大家不再像农业社会那样只是顾好自己的田然后交税就可以了，而是更渴望对国家的各类制度、政府行为有更多了解。

其实我觉得不管是陈丹青、老舍还是许子东，或者鲁迅，或者高华，或者其他学者名人，或者是像 R 江湖的大佬我们共同的朋友益辉，任何人都无法帮我们指出我们自己的人生之路。找他们借来的答案大多都没法刚好迁移应用到我们自己身上，他们每个人都有自身的局限。当然啦，我不是说要放弃找他们借答案，像你写的这篇博客就是你找他们借来的东西经过自己咀嚼后思考得到的感悟。我觉得你写得很好，继续加油吧。

我倒觉得，现在不是读书难得，而是独立思考更难得，毕竟上网随便搜搜就能借来许多别人的详细答案。

**Reply by XiangyunHuang** [reply]
> 这个名列前茅的十大是哪个榜单，我完全没听说过这部小说

十多年前的事了，当时在学校图书馆第一次看到这本书，看完之后，在网上搜了一下，就看到这么个说法，并不是什么权威评选结果，就好像知乎里评选年度最佳xxx。

> 原来，现在纯粹的读书和几个世纪前一样，都是贵族的娱乐。

我说这句话当时想到的意思是两三个世纪前，那些研究数学、生物、统计的人，大多是贵族，他们不为普通的生活所累，即使为了五斗米，那他们在追求科学艺术的道路上也纯粹得多，现在看到的搞艺术其实是搞钱。

**Comment by botsp** [comment]
大致想了想，可能老舍是不爱读书的我读过最多的作家了。
故事讲的那样好，语言讲的那样引人，主题是那样地关切现实世界。祥子大概也是那个社会的边缘人群吧，尽管上海北京的街头都是他们和他们的车，但对于那个时代进程来说，似乎真的不重要。从来不重要，当时不重要，现在也不重要。

我爱老舍，古诗只读杜甫的、小说只读老舍的。

**Reply by XiangyunHuang** [reply]
《四世同堂》读过没？我读完《骆驼祥子》和《离婚》后，想挑战大部头的，结果没一会儿就卡住了。一些学者说《四世同堂》蛮好，但是我却读不下去了。你有啥感触么？再激励激励我。

**Reply by botsp** [reply]
俺也没读完，可能是心境上还读不了长篇；也感觉那种大家庭的叙事框架吸引不了我。

`我这一辈子`或者`月牙`这种，家庭只是社会结构的其中一个单元，在时代面前支离破碎又勉强挣扎，我可能是更喜欢这种故事。他们的家、租一个四合院的里屋真就是家了，更多的故事是在家以外发生的。家人是在这个世界流浪的伙伴，在此之间的矛盾纠纷或者温情，都只是偶尔的，都是宏大世界的一条支线，但也都能让我有感受到。

四世同堂给我一种感觉，是社会伦理先定义好家庭身份，然后再去演绎彼此的矛盾纠葛。当然我没有读完四世同堂，这只算是我对四世同堂的想象吧，在我的偏见里他们和雷雨一样属于大家庭。不过同为老舍爱好者，争取有一天能读完这本~ 

`离婚`我读完了，之前还设想那样的故事在今天会是怎样的，感觉应该不太可能发生了，连老李的精神苦闷在今天都显得很珍贵吧。

**Reply by XiangyunHuang** [reply]
对《四世同堂》，有的学者说，它也服从那个年代的左翼套路，大哥是传统的知识分子，是优柔寡断的，瞻前顾后的，也是负责任的，也是爱国的，忍辱负重地爱，既要为老的着想，又要为小的考虑。而小的，不管那么多，很有革命性，不管不顾离家出走，找革命队伍。中间的那个比较洋派，精致的利己主义者。我看到了老舍对大哥纠结的心理进行细腻的描写，也看到了中间那个为了自己的利益挖空心思到处巴结人，甚至无底线扯上日本人的关系。书的部分，我目前连第一部《惶惑》都没看完，只看到日本人进城，小羊圈胡同里，大家聚在一起帮助钱家治病、救人、埋人。

**Reply by XiangyunHuang** [reply]
丁聪的插图、老舍画的小羊圈胡同的手稿，在美术馆我都看到了，等我把书看完，也写篇读后感，Flag 先立起来。

**Reply by botsp** [reply]
期待! (●'◡'●)
